web_responsive/static/src/legacy/js/web_responsive.js

```diff
@@ -169,6 +169,7 @@ odoo.define("web_responsive", function (require) {
     ViewDialogs.SelectCreateDialog.include({
         init: function () {
             this._super.apply(this, arguments);
+            this.viewType = "kanban";
```

Comment:
This should only happen on mobile phones
```suggestion
            if (config.device.isMobile) {
                    this.viewType = "kanban";
            }
```

Reply:
> There are some defects (not related to this PR)

Thank you! I've found problem with redundant padding on kanban card. But I can't reproduce situation with controls on the first screenshot. Could you please describe steps to reproduce?

> This should only happen on mobile phones

Shame on me. Fixed.